source/blog/2018-11-29-december-event.md

```diff
@@ -0,0 +1,37 @@
+---
+title: First Annual DecEmber Event!
+author: Melanie Sumner
+tags: Recent Posts, 2018
+responsive: true
+---
+
+December is a great month to support your friendly Ember.js learning team!
+
+Maybe December is a busy month for you, because of the holidays and family obligations- and that’s totally okay- we hope you have a great time! But for some, December is a time of the year with a lot of downtime. Maybe that’s you! Are you wondering what to do with all your vacation time in December? We’ve got ya covered!
+
+This year, the Ember Learning Core Team is sponsoring our first annual DecEmber, a hackathon event specifically targeted at Learning Team efforts. For an extra bit of awesome, it’s running at the same time as [24 Pull Requests](https://24pullrequests.com/), so contributions to DecEmber will count for both!
+
+## Participation Rules
+
+To qualify for any of our random assortment of thank you goodies (that we’ve delightfully dubbed Embercouragement), you must complete at least 5(five) participation activities for repositories in the [ember-learn org](https://github.com/ember-learn) between December 1 and December 28, 2018.
+
+## Participation Activities
+
+- Submitting PRs for issues
+  - in the [Ember Learn Org](https://github.com/ember-learn)
+  - that have the `DecEmber` label
+- Writing Issues for repositories in the ember-learn org. We will mark them with the `DecEmber` label if they qualify.
+- Creating Illustrations: Maybe you have an artistic side, and illustrating code concepts is fun for you! If that’s the case, there are always illustrations that would enhance the guides, and those are very welcome for this event!
+- If you have another idea that you think would help support the Learning Core Team- let us know!
+
+## Quality Standards
+
+We don’t think we need to say these things, but just in case….
```

Comment:
```suggestion
We don’t think we need to say these things, but just in case…
```

I'd consider deleting this line all together, it's almost a code of conduct for the event?